srcpkgs/avr-binutils/template

```diff
@@ -3,43 +3,25 @@ pkgname=avr-binutils
 version=2.32
 revision=1
 wrksrc="binutils-$version"
+build_style=gnu-configure
+configure_args="--disable-werror --enable-shared --enable-gold
+ --enable-plugins --enable-install-libbbfd --build=x86_64-unknown-linux-gnu
+ --host=${XBPX_CROSS_TRIPLET} --target=avr --with-pic"
+make_install_args="tooldir=/usr"
+makedepends="zlib-devel"
 short_desc="GNU binary utilities for AVR"
 maintainer="allan <mail@may.mooo.com>"
-license="GPL-3"
+license="GPL-3.0-only"
 homepage="http://www.gnu.org/software/binutils/"
 distfiles="$GNU_SITE/binutils/binutils-$version.tar.xz"
 checksum=0ab6c55dd86a92ed561972ba15b9b70a8b9f75557f896446c82e8b36e473ee04
 
-makedepends="zlib-devel"
-
 do_configure() {
-	./configure \
-		--disable-werror \
-		--enable-ld=default \
-		--enable-shared \
-		--enable-gold \
-		--enable-plugins \
-		--enable-threads \
-		--enable-install-libbfd \
-		--prefix=/usr \
-		--build=x86_64-unknown-linux-gnu \
-		--host=$XBPS_CROSS_TRIPLET \
-		--target=avr \
-		--with-pic
-
-	make configure-host
-}
-
-do_build() {
-	make tooldir=/usr
-}
-
-do_install() {
-	make prefix="$DESTDIR"/usr tooldir="$DESTDIR"/usr install
+	./configure ${configure_args}
```

Comment:
I believe since you put everything in the `configure_args` you can just as well remove `do_configure()`.